afrikaans/beskuit/beskuit.md

```diff
@@ -0,0 +1,10 @@
+Jy voel jy sal graag
+- [Ouma](./ouma/ouma.md) winkel beskuit
+- [Tuisgebakte] (./tuisgebakte/tuisgebakte.md) beskuit
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.